com.coala.core/plugin.xml

```diff
@@ -57,7 +57,7 @@
       <menuContribution
             locationURI="popup:org.eclipse.ui.popup.any?after=additions">
          <menu
-               icon="icons/coala-analyzer.gif"
+               icon="icons/coala.gif"
```

Comment:
You have to rename this file https://github.com/coala/coala-eclipse/blob/master/com.coala.core/icons/coala-analyzer.gif to `coala.gif`